js/hubs/components/MediaDropzone.js

```diff
@@ -18,7 +18,7 @@ const MediaDropzone = ({ type, setArtwork, setTrack, disabled }) => {
           alert(`Your track is not an MP3. \nPlease upload an MP3.`);
         } else {
           alert(
-            `your track is ${size} mb... \nPlease upload a smaller than 110 mb`
+            `your track is ${size} mb... \nPlease upload a smaller than 500 MBs`
```

Comment:
We should use config-like constants wherever possible instead of hardcoded values like this.